researcher_ids/RI_Claims_V1.md

```diff
@@ -60,90 +65,85 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+    as per the "iss" field.
 
--   Encoded as a JSON object that contains various properties
-    ([Claim Object Fields](#claim-object-fields)) that describe the assertion and
+-   Encoded as an
+    [Embedded Token](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#term-embedded-token)
+    that contains various properties
+    ([RI Claim Object Fields](#ri-claim-object-fields)) that describe the assertion and
     limitations thereof.
+    
+-   RI Claim Embedded Tokens are encoded as strings and included as a list of
+    strings returned within a [RI Claim](#researcher-identity-claim-ri-claim). See
+    the [RI Claim Embedded Token](#ri-claim-embedded-token) section of this
+    specification for more details.
 
-#### **Claim Authority**
+#### **Researcher Identity Claim Object ("RI Claim Object")**
 
--   The
-    [AAI Claim Authority](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#term-claim-authority)
-    as encoded by the [source field](#source-required).
+An OIDC claim "ga4gh_rio" that contains fields that provide GA4GH
+standard information about a claim that is not captured in the standard
+set of claims published by underlying OIDC specification.
 
-#### **Passport**
+For field definitions, refer to [RI Claim Object Fields](#ri-claim-object-fields).
 
--   A bundle of
-    [researcher identity claims](#researcher-identity-claim-ri-claim) that is
-    collected into a
-    [specialized JWT token](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#ga4gh-jwt-format)
-    and signed by an
-    [Identity Broker](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#term-identity-broker)
-    as per the
-    [GA4GH AAI specification](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md)
-    for the purpose of encoding identity and evaluating authorization.
+#### **RI Claim Object Identity**
 
--   The bundle of claims includes any indirect RI Claims that such a JWT token
-    makes available to the caller of the corresponding /userinfo endpoint, as
-    well as any RI Claims that may be collected either directly on JWTs provided
-    within the "ga4gh_passports" object or within responses to the /userinfo
-    endpoints for the "ga4gh_passports" tokens.
+-   The {"iss", "sub"} pair of OIDC standard claims that are included
+    within an [RI Claim Object](#researcher-identity-claim-object) that
+    represents a given user (such as a user account) within the "iss"
+    system.
 
--   The bundle of claims includes all RI Claims made available to recursive
-    calls to /userinfo following other embedded Passport tokens that are present
-    within its claims.
+#### **Claim Source**
 
-#### **Embedded Passport Token**
+-   The
+    [AAI Claim Source](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#term-claim-source)
+    as encoded by the [source field](#source).
+
+#### **Passport**
```

Comment:
I know that a "Passport" is a somewhat conceptual idea, though its presence in the specifications always feels like it should be a specific data object with a specific data structure. I think from a data structure perspective we're referring to a bundle of claims as a token? Should we reconcile that language somewhat?

Reply:
Although probably not what you were expecting, there has been a large overhaul of definitions including this one, as well as an Overview section to try to tie it together better.